_episodes/62-cd.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Continuous Delivery/Continuous Deployment with BitBucket Pipelines"
+teaching: 15
+exercises: 30
+questions:
+- "How can I set up Continuous Deployment/Continuous Delivery in BitBucket Pipelines?"
+objectives:
+- "Understand how to set up Continuous Deployment/Continuous Delivery in BitBucket Pipelines"
+keypoints:
+- "You can configure automated deployment in the `bitbucket-pipelines.yml` file"
+---
+## Setting up a BitBucket Pipeline to deploy an application
+To understand how to set up Continuous Delivery or Continuous Deployment with BitBucket Pipelines we will 
+look at the BitBucket Pipeline template to deploy a React application to AWS.
+
+You do not need to understand the specific details of react, node.js or AWS. It is more important that you understand
+the general setup of such a deployment pipeline. The same kind of setup can be used for any application written in any 
+language deployed to any on-premise or cloud server.
+
```

Comment:
To make it look less scary, we could already add here
```suggestion
To keep secrets like SSH keys out of public files, they can be stored in [repository variables](https://support.atlassian.com/bitbucket-cloud/docs/variables-and-secrets/#Variablesinpipelines-User-definedvariables)
```

Reply:
Nice!